### ISSUE TITLE
Logged out / projects removed on restart app

### ISSUE THREAD

**Issue by bcallaars** [cypress-io/cypress#66]
Whenever I restart the desktop app, I'm logged out again and all my projects are removed.

Console output:
```
carl-the-llama% [8129:0925/173301:WARNING:x11_util.cc(1490)] X error received: serial 11738, error_code 3 (BadWindow (invalid Window parameter)), request_code 4, minor_code 0 (X_DestroyWindow)
Failed to tell parent about crash.
~/.cypress/Cypress/Cypress
[28603:0925/173303:ERROR:browser_main_loop.cc(170)] Running without the SUID sandbox! See https://code.google.com/p/chromium/wiki/LinuxSUIDSandboxDevelopment for more information on developing with the sandbox on.
ATTENTION: default value of option force_s3tc_enable overridden by environment.
```

**Comment by brian-mann** [comment]
Also noting this is on Ubuntu Desktop.

That console output is actually fine, there's nothing in there - there is most likely a permission issue with persisting the Cypress settings. Those aren't being saved and whenever the Desktop App is reopened it's requiring you to login + add projects again.

**Comment by brian-mann** [comment]
Fixed in [`0.11.7`](https://github.com/cypress-io/cypress/wiki/Changelog#0117-09252015).
